### DISCUSSION TITLE
Question About The Mod

### DISCUSSION THREAD

**Discussion by RawrOne** [MAFINS/MenyooSP#435] [Q&A]
is Menyoo will not work today because of the Update?

**Comment by yomamadas** [comment]
I am staying with 1.0.2372, too much problems with the newer versions
if you want to play with the new version, make a copy of the whole gtaV and update the original.
When things turn south, just rename the copy and rename the original into copy and you have the best of 2 worlds

![2022-12-14_173405](https://user-images.githubusercontent.com/119436721/207666805-6ee0af20-129a-4cef-b99a-55ab214e5ed6.jpg)
